docs/usage/configuration-options.md

```diff
@@ -2464,7 +2464,7 @@ Here's an example config to limit the "noisy" `aws-sdk` package to weekly update
   "packageRules": [
     {
       "matchPackageNames": ["aws-sdk"],
-      "schedule": ["after 9pm on sunday"]
+      "schedule": ["* 21-23 * * 7"]
```

Comment:
@rarkins should be we add a human readable description?

so this is effective 21:00 - 23:59

Reply:
Add where?

Reply:
yes

Reply:
We already explain the schedule in the above statement.